src/Algebra/Graph/Class.hs

```diff
@@ -395,3 +399,8 @@ forest = overlays . map tree
 class ToGraph t where
     type ToVertex t
     toGraph :: (Graph g, Vertex g ~ ToVertex t) => t -> g
+
+class Escape s where
+    escape :: s -> s
+instance Escape String where
+    escape = concatMap (\x-> if x == '"' then "\\\"" else [x])
```

Comment:
Since this is DOT-specific escaping, I suggest to move this class to `Export.Dot`.